### DISCUSSION TITLE
Winning agents from the competition

### DISCUSSION THREAD

**Discussion by jacksondc** [yasserfarouk/scml#76] [General]
Hi Yasser! Have the winning OneShot agents been released anywhere yet? We'd love to keep testing against them. (And have the write-ups been released?)

**Comment by yasserfarouk** [comment]
Yes they are on 
[https://www.GitHub.com/yasserfarouk/scml-agents](https://www.GitHub.com/yasserfarouk/scml-agents)

You can install them all using

```
pip install scml-agents
```

**Reply by jacksondc** [reply]
Thank you!
